spec/omniauth/auth0/jwt_validator_spec.rb

```diff
@@ -10,6 +10,7 @@
   let(:client_id) { 'CLIENT_ID' }
   let(:client_secret) { 'CLIENT_SECRET' }
   let(:domain) { 'samples.auth0.com' }
+  let(:issuer) { 'samples.auth0.com' }
```

Comment:
if the issuer is the same as the domain, how can the test check that it changed?